lib/std/os.zig

```diff
@@ -4127,7 +4131,7 @@ pub fn realpathW(pathname: []const u16, out_buffer: *[MAX_PATH_BYTES]u8) RealPat
 /// Return canonical path of handle `fd`.
 /// This function is very host-specific and is not universally supported by all hosts.
 /// For example, while it generally works on Linux, macOS or Windows, it is unsupported
-/// on FreeBSD, or WASI.
+/// on WASI. FreeBSD has an inefficient alternative.
```

Comment:
It's not even a matter of efficiency here, the function can fail depending on how many open files you have. I'd just mark this as unsupported and spare the users a huge headache when they'll have to debug this mysterious failure (or require an allocator).

Reply:
Sure thing, I'll just leave that as the existing behaviour then. Thanks for your time!